msg/VehicleThrustSetpoint.msg

```diff
@@ -3,3 +3,6 @@ uint64 timestamp        # time since system start (microseconds)
 uint64 timestamp_sample # timestamp of the data sample on which this message is based (microseconds)
 
 float32[3] xyz          # thrust setpoint along X, Y, Z body axis [-1, 1]
+
+# TOPICS vehicle_thrust_setpoint_0 vehicle_thrust_setpoint_1
```

Comment:
Instead of this breaking change, could you use topic instances instead?
And if we eventually combine the controls, we can avoid another change then.

Reply:
Very true ofc, I don't remember why I changed it, reverted with e8ccbf9eb2462504ae5e24b71e6f849f18c51fb5.